tests/rules/deprecation.test.ts

```diff
@@ -459,6 +491,26 @@ ruleTester.run('deprecation', rule, {
       const def4: Interface1 = {};    // ERROR: Interface1
       const component = <Component/>  // ERROR: Component
       `),
+    getInvalidTestCase(`
+      interface Interface {
+        method(param: string): void;
+        /** @deprecated */ method(): void;
+        /** @deprecated */ method(param: number): void;
+      }
+      const obj: Interface = { method(args: any) {} };
+      obj.method();  // ERROR: method
+      obj.method(1); // ERROR: method
+    `),
+    getInvalidTestCase(`
+      class Class {
+        method(param: string): void;
+        /** @deprecated */ method(): void;
+        /** @deprecated */ method(param: number): void;
+      }
+      const obj = new Class();
+      obj.method();  // ERROR: method
+      obj.method(1); // ERROR: method
```

Comment:
Please add check that non-deprecated signature is not errored.
```suggestion
      obj.method('valid');  // missed
      obj.method();  // ERROR: method
      obj.method(1); // ERROR: method
```